samples/subsys/usb_c/sink/src/main.c

```diff
@@ -16,7 +16,8 @@ LOG_MODULE_REGISTER(main, LOG_LEVEL_DBG);
 #define PORT1_NODE DT_NODELABEL(port1)
 #define PORT1_POWER_ROLE		DT_ENUM_IDX(DT_NODELABEL(port1), power_role)
 
-#if (PORT1_POWER_ROLE != TC_ROLE_SINK)
+/* A Sink power role evauates to 0. See usbc_tc.h: TC_ROLE_SINK */
+#if (PORT1_POWER_ROLE)
```

Comment:
Why is this change needed? This check will fail if the `enum tc_power_role` options change.

Reply:
I am also curious about this.

Reply:
I broke something and thought this was the problem but it's okay. I'm going to abandon this PR.